contracts/contracts.go

```diff
@@ -51,6 +51,27 @@ func ExampleToken(fungibleTokenAddr string) []byte {
 	return []byte(code)
 }
 
+// CustomizableExampleToken returns the ExampleToken contract with a custom Name.
+//
+// The returned contract will import the FungibleToken contract from the specified address.
+func CustomizableExampleToken(fungibleTokenAddr, tokenName string) []byte {
```

Comment:
```suggestion
func CustomToken(fungibleTokenAddr, tokenName string) []byte {
```

Maybe just shorten it for simplicity?